server/plugins/OscUGens.cpp

```diff
@@ -1414,17 +1414,45 @@ force_inline void Osc_ika_perform(OscType *unit, const float * table0, const flo
 
 	int32 freq = (int32)(unit->m_cpstoinc * freqin);
 	float radtoinc = unit->m_radtoinc;
-	LOOP1(inNumSamples,
+
+	int unrolled4 = inNumSamples / 4;
+	int remain4   = unrolled4 - unrolled4 * 4;
```

Comment:
Should this perhaps be `int remain4   = inNumSamples - unrolled4 * 4;` ? You might want to check empirically that an unusual blocksize is working ok.